app/assets/javascripts/sufia/save_work/save_work_control.es6

```diff
@@ -95,6 +96,13 @@ export class SaveWorkControl {
   validateFiles() {
     if (!this.isNew || this.uploads.hasFiles) {
       this.requiredFiles.check()
+
+      if (this.uploads.hasFiles && this.mustAgreeAgain) {
+        // Force the user to agree before submitting
+        this.form.find('input[name="agreement"]').prop("checked", false)
```

Comment:
Can you deal with this at a higher level? `this.depositAgreement.setNotAccepted` perhaps?